bluepysnap/config.py

```diff
@@ -163,11 +164,57 @@ def to_dict(self):
 class CircuitConfig(Config):
     """Handle CircuitConfig."""
 
+    def __init__(self, *args):
+        """Initializes circuit config."""
+        super().__init__(*args)
+        self._populations = self._resolve_population_configs(self.to_dict())
+
     @classmethod
     def from_config(cls, config_path):
         """Instantiate the config class from circuit configuration."""
         return cls(config_path, libsonata.CircuitConfig)
 
+    @property
+    def node_populations(self):
+        """Access node population configs."""
+        return self._populations["nodes"]
+
+    @property
+    def edge_populations(self):
+        """Access edge population configs."""
+        return self._populations["edges"]
+
+    # NOTE: These are parsed here to keep Parser returning the exact parsed configuration.
```

Comment:
I don't understand this comment 🤔 
Can it be clarified and moved into the relevant method, or removed?

Reply:
Yeah, I removed the comment as it was more confusing than informative.

Short story time: I was initially thinking of parsing the population configs in the `class Parser` and returning them as extra parts in the config, but then realized it may or may not break stuff and/or make stuff more confusing so I added a specific method to `class CircuitConfig`. Hence the comment.